tools/test-proxy/Azure.Sdk.Tools.TestProxy/Startup.cs

```diff
@@ -56,97 +55,52 @@ private static string resolveRepoLocation(string storageLocation = null)
-            if (args[0].StartsWith("-"))
-            {
-                return;
-            }
-
-            // last but not least, the first argument is a verb, verify it's our verb
-            // version and help are default verbs and need to be in here
-            string[] array = { "start", "reset", "restore", "push", "version", "help" };
-            if (!array.Contains(args[0]))
-            {
-                // The odd looking formatting is to make this look like the same error
-                // CommandLineParser would output if the verb wasn't recognized.
-                string error = @$"ERROR(S):
-  Verb '{args[0]}' is not recognized.
-
-  --help       Display this help screen.
-
-  --version    Display version information.
-";
-                System.Console.WriteLine(error);
-                Environment.Exit(1);
-            }
+            Environment.Exit(resultCode);
         }
 
         private static async Task Run(object commandObj)
         {
-            new GitProcessHandler().VerifyGitMinVersion();
-            DefaultOptions defaultOptions = (DefaultOptions)commandObj;
-
             var assembly = System.Reflection.Assembly.GetExecutingAssembly();
             var semanticVersion = assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
             System.Console.WriteLine($"Running proxy version is Azure.Sdk.Tools.TestProxy {semanticVersion}");
 
+            new GitProcessHandler().VerifyGitMinVersion();
+            DefaultOptions defaultOptions = (DefaultOptions)commandObj;
+
             TargetLocation = resolveRepoLocation(defaultOptions.StorageLocation);
             Resolver = new StoreResolver();
             DefaultStore = Resolver.ResolveStore(defaultOptions.StoragePlugin ?? "GitStore");
+            var assetsJson = string.Empty;
 
             switch (commandObj)
             {
+                case ConfigLocateOptions configOptions:
+                    assetsJson = RecordingHandler.GetAssetsJsonLocation(configOptions.AssetsJsonPath, TargetLocation);
+                    System.Console.WriteLine(await DefaultStore.GetPath(assetsJson));
+                    break;
+                case ConfigShowOptions configOptions:
+                    assetsJson = RecordingHandler.GetAssetsJsonLocation(configOptions.AssetsJsonPath, TargetLocation);
+                    using(var f = File.OpenRead(assetsJson))
+                    {
+                        using var json = JsonDocument.Parse(f);
+                        System.Console.WriteLine(JsonSerializer.Serialize(json, new JsonSerializerOptions { WriteIndented = true }));
+                    }
+                    break;
+                case ConfigCreateOptions configOptions:
+                    assetsJson = RecordingHandler.GetAssetsJsonLocation(configOptions.AssetsJsonPath, TargetLocation);
+                    throw new NotImplementedException("Interactive creation of assets.json feature is not yet implemented.");
```

Comment:
You could link to the transition script doc as well here as an intermediate step, just to help discoverability.